src/main/native/cpp/view/AnalyzerPlot.cpp

```diff
@@ -278,4 +293,18 @@ void AnalyzerPlot::DisplayTimeDomainPlots() {
       }
     }
   }
+  return true;
+}
+
+bool AnalyzerPlot::LoadPlots() {
+  // See if the plots are loaded
+  bool graphLoaded = DisplayVoltageDomainPlots() && DisplayTimeDomainPlots();
+
+  // The plots have already been loaded if its currently loaded and it was
+  // previously loaded
+  bool alreadyLoaded = graphLoaded && m_prevGraphLoaded;
+
+  // Update previously loaded value
+  m_prevGraphLoaded = graphLoaded;
+  return alreadyLoaded;
```

Comment:
Checking m_prevGraphLoaded is redundant since we only care about the current value, not whether it changed since the last value.

Reply:
The issue is we have to set the scroll bar position just after the graphs have loaded.

Reply:
Right, but this code checks when both variables return true, not when one of them goes from false to true.

Reply:
Then should we keep the `m_prevgraphLoaded` as a member of the Analyzer?